taf/docker-compose-taf-mqtt-bus.yml

```diff
@@ -1127,12 +1127,17 @@ services:
     - database
     - mqtt-broker
     environment:
-      EDGEX__DEFAULT__OPTIONAL__CLIENTID: kuiper-rules-engine
-      EDGEX__DEFAULT__PORT: 1883
-      EDGEX__DEFAULT__PROTOCOL: tcp
-      EDGEX__DEFAULT__SERVER: edgex-mqtt-broker
+      CONNECTION__EDGEX__MQTTMSGBUS__OPTIONAL__CLIENTID: kuiper-rules-engine
+      CONNECTION__EDGEX__MQTTMSGBUS__PORT: 1883
+      CONNECTION__EDGEX__MQTTMSGBUS__PROTOCOL: tcp
+      CONNECTION__EDGEX__MQTTMSGBUS__SERVER: edgex-mqtt-broker
+      CONNECTION__EDGEX__MQTTMSGBUS__TYPE: mqtt
+      CONNECTION__EDGEX__REDISMSGBUS__PORT: 6379
+      CONNECTION__EDGEX__REDISMSGBUS__PROTOCOL: redis
+      CONNECTION__EDGEX__REDISMSGBUS__SERVER: edgex-redis
+      CONNECTION__EDGEX__REDISMSGBUS__TYPE: redis
+      EDGEX__DEFAULT__CONNECTIONSELECTOR: edgex.mqttMsgBus
```

Comment:
Note that since the ENV vars in the base are different names, they don't get replaced. Key here is `EDGEX__DEFAULT__CONNECTIONSELECTOR: edgex.mqttMsgBus` and the addition of `CONNECTION__EDGEX__MQTTMSGBUS__*` env vars

Reply:
@jinlinGuan I expected the smoke tests to fail for the tests using MQTT MessageBus per my note above, but they didn't.

Reply:
The current kuiper testcases only use Redis MessageBus. Should we modify the test scripts to run with both MessageBus?